README.md

```diff
@@ -1,31 +1,654 @@
-# Module2 Project
+# Module-2 Project
```

Comment:
Suggestion (because I think this also helps the instructors), provide a lab or lesson after the section you think each task should be implemented, reminding students to work on their project. For example, task 1 goes after methods and task 2 goes after multiple classes and task 3 goes at the end. In each lab/lesson re-iterate the tasks they should/could do so they have time to work on it.

Reply:
OK, that's a good idea.